core/templates/pages/exploration-editor-page/services/exploration-diff.service.ts

```diff
@@ -285,8 +285,8 @@ export class ExplorationDiffService {
           change.cmd !== 'AUTO_revert_version_number' &&
           change.cmd !== 'edit_exploration_property' &&
           change.cmd !== 'add_written_translation' &&
-          change.cmd !== 'mark_written_translation_as_needing_update' &&
-          change.cmd !== 'mark_written_translations_as_needing_update'
+          change.cmd !== 'mark_translations_as_needing_update' &&
+          change.cmd !== 'remove_translations'
```

Comment:
Should we keep the old cmds too (in case there are some old versions using the cmds)? Not sure.

Reply:
I was just thinking of visualising / comparing old versions containing the deprecated commands.